requirements.txt

```diff
@@ -5,3 +5,4 @@ cython>=0.23
 coverage>=4.0
 six>=1.10
 wheel>=0.29
+lxml>=3.6.0
```

Comment:
I don't think we want to eat this dependency on the whole of grpc for the sake of a unit test. We'd rather add it to the list of pip installed packages into the docker files.